app/graphql/mutations/system/associate_profiles.rb

```diff
@@ -6,6 +6,8 @@ module System
     class AssociateProfiles < BaseMutation
       graphql_name 'associateProfiles'
 
+      enforce_rbac Rbac::COMPLIANCE_ADMIN
```

Comment:
Why is this `COMPLIANCE_ADMIN` while the reverse mapping is `POLICY_WRITE` at `associate_systems`?